### ISSUE TITLE
(2.0) FGlob ignore trick does not work in tundra2 (path changes?)

### ISSUE THREAD

**Issue by leidegre** [deplinenoise/tundra#170]
Previously, I've relied on the ability to do this:

```
FGlob { 
    Dir = "src", 
    Extensions = { ".cpp" },
    Filters = {
        { Pattern = "/_[^/]*$"; Config = "ignore" },
    },
},
```

The side-effect of this is that tundra will ignore files that start with `_` underscore, however, this does not seem to carry over in tundra2.

I can't see that anything has changed in `glob.lua` does tundra2 represent the path in a different way (normalized somehow) that causes this match to fail (note the leading forward slash)?

If I remove the leading forward slash, it appears to be working. I used to do that, to anchor the pattern at the beginning of the file name, why would that stop working in tundra2?

In summary, this `/_[^/]*$` does not match, this `_[^/]*$` does (note the leading forward slash).
